ar/index.md

```diff
@@ -0,0 +1,7 @@
+---
+layout: default
+language: ar
+direction: rtl
+---
+
+{% include chapters.html path = "/en/" %}
```

Comment:
Please change this into `{% include chapters.html %}` as stated in: https://github.com/HugoGiraudel/sass-guidelines/blob/gh-pages/TRANSLATING.md.